.github/workflows/main.yml

```
@@ -35,3 +35,52 @@ jobs:
         uses: SonarSource/sonarqube-scan-action@v5
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+  version-bump:
+
+      name: Bump Version
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)